### PR TITLE
Add shardingsphere-scaling-elasticjob module

### PR DESCRIPTION
**Pull request by Lucas-307**

### CODE REVIEW

**Comment by Lucas-307** [comment]
please wait for ci.

File: shardingsphere-scaling/shardingsphere-scaling-elasticjob/pom.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            <artifactId>elasticjob-lite-lifecycle</artifactId>
+            <version>${elasticjob.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>${logback.version}</version>
+        </dependency>
+    </dependencies>
+</project>
```
Comment (dongzl) [inline shardingsphere-scaling/shardingsphere-scaling-elasticjob/pom.xml:58]: blank line at end of file.

Comment (dongzl) [inline shardingsphere-scaling/shardingsphere-scaling-elasticjob/pom.xml:58]: blank line at end of file.

Comment (Lucas-307) [reply shardingsphere-scaling/shardingsphere-scaling-elasticjob/pom.xml]: fixed

**Review by dongzl — APPROVED** [review]

**Review by tristaZero — APPROVED** [review]

**Pull request merged**